torch/_torch_docs.py

```diff
@@ -1785,6 +1785,40 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.clamp`.
 """.format(**common_args))
 
+add_docstr(torch.column_stack,
+           r"""
+column_stack(tensors, *, out=None) -> Tensor
```

Comment:
The doc build failure is real although I don't see what's causing it. You may need to try building the documentation locally (see https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md#building-documentation) to identify the source of the error. Sometimes I do this by building, deleting the built documentation, changing the doc string, then rebuilding. Bisecting the documentation this way usually quickly reveals the source of the error, although its cause may be inscrutable.

Reply:
Thank you so much for providing the documentation link and sharing the experience of bisecting documentation errors.
I updated the function description and built the documentation successfully on my local machine.